src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 20;
+    private static final Random random = new Random();
+
+    public Figure[] getRandomArrayOfFigures() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        Figure[] figures = new Figure[figureNumber];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = getRandomFigure();
+        }
+        return figures;
+    }
```

Comment:
lets move this logic into main method

Reply:
Moved this code to Main class in main method